packages/venia-ui/lib/components/AccountMenu/accountMenu.css

```diff
@@ -0,0 +1,31 @@
+.root {
+    background-color: rgb(var(--venia-global-color-background));
+    border-radius: 1px;
+    box-shadow: 1px 1px 5px rgb(var(--venia-global-color-gray-darker));
+    display: grid;
+    grid-auto-flow: row;
+    justify-content: stretch;
+    opacity: 0;
+    padding: 1.5rem 2.5rem;
+    position: fixed;
+    right: -100vw;
+    row-gap: 1.5rem;
+    top: 5rem;
+    transform: translate3d(0, -8px, 0);
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-global-anim-out);
+    transition-property: opacity, transform, visibility;
+    visibility: hidden;
+    width: 27.5rem;
+}
+
+.root_open {
+    composes: root;
```

Comment:
I have been removing the space after `composes` in the mini cart PR's, thinking someone added it by mistake. Apparently not 🤣. Good to separate composes from other attributes.

Reply:
Haha no worries. Yeah I don't know, I just didn't want it to get lumped in with regular CSS properties.